vars/artifactsPipeline.groovy

```diff
@@ -0,0 +1,155 @@
+#! groovy
+
+def call(Map pipelineParams) {
+    pipeline {
+        agent {
+            label {
+                label getJenkinsLabels(params.backend, 'eu-west-1')
+            }
+        }
+        environment {
+            AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
+            AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
+        }
+        parameters {
+            string(defaultValue: "${pipelineParams.get('backend', 'gce')}",
+                   description: 'aws|gce',
+                   name: 'backend')
+            string(defaultValue: '',
+                   description: 'a Scylla repo to run against',
+                   name: 'scylla_repo')
+            string(defaultValue: "${pipelineParams.get('test_config', 'test-cases/artifacts/centos7.yaml')}",
+                   description: 'a config file for the artifacts test',
+                   name: 'test_config')
+             string(defaultValue: "${pipelineParams.get('post_behavior_db_nodes', 'keep-on-failure')}",
+                   description: 'keep|keep-on-failure|destroy',
+                   name: 'post_behavior_db_nodes')
```

Comment:
Need to add instance_provision to the pipeline.
We may need to change it when spot instances aren't available.

Reply:
I've added this parameter.